SampleCode/SwiftSampleCode/DronecodeSDKSwiftDemo/Models/ExampleMission.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  ExampleMission.swift
+//  DronecodeSDKSwiftDemo
+//
+//  Created by Marjory Silvestre on 26.04.18.
+//  Copyright © 2018 Marjory Silvestre. All rights reserved.
+//
+
+import Foundation
+import Dronecode_SDK_Swift
+
+
+class ExampleMission {
+    
+    var missionItems = [MissionItem]()
+
+    init(){
+        
+        if missionItems.isEmpty {
+            missionItems = [MissionItem]()
+            
+            do {
+                let entry = MissionItem(latitudeDeg: 47.398039859999997, longitudeDeg: 8.5455725400000002, relativeAltitudeM: 10.0, speedMPS: 2.0, isFlyThrough: true, gimbalPitchDeg: -60.0, gimbalYawDeg: -90.0, cameraAction: CameraAction.START_PHOTO_INTERVAL)
```

Comment:
This reminds me that I should check how it works. Ideally, if you don't set one of the fields, it should take the previous one. Which means that you would not have to set the same speed on all items.

I'll have a look at that, later.

Reply:
Correct, if it is set to `NaN` it should not add the mavlink mission item to set a(nother) speed.